backend/battle/urls.py

```diff
@@ -1,18 +1,20 @@
 from django.urls import path
 from .views import (
     Home,
-    # RoundNewCreator,
-    # RoundNewOponnent,
-    # Invite,
-    # Opponent,
+    BattleView,
+    Invite,
+    TeamView,
+    BattleList,
+    BattleDetail,
 )
 
 
 urlpatterns = [
     path("", Home.as_view(), name="home"),
-    # path('round/new/', RoundNewCreator.as_view(), name='round_new'),
-    # path('invite/', Invite.as_view(), name='invite'),
-    # path('opponent/', Opponent.as_view(), name='opponent'),
-    # path('player2/round', RoundNewOponnent.as_view() , name='round_new2'),
+    path("battle/", BattleView.as_view(), name="battle"),
+    path('invite/', Invite.as_view(), name='invite'),
+    path('battle/list', BattleList.as_view(), name='battle_list'),
+    path("team/<int:pk>/create/<int:user>", TeamView.as_view(), name="team_create"),
```

Comment:
You don't need to pass the user here. You can use the logged user inside the view